lib/ssl/test/ssl_api_SUITE.erl

```diff
@@ -2638,6 +2641,19 @@ invalid_options_tls13(Config) when is_list(Config) ->
           end,
     [Fun(Option, ErrorMsg, Type) || {Option, ErrorMsg, Type} <- TestOpts].
 
+
+ssl_not_started() ->
+    [{doc, "Test that an error is returned if ssl is not started"}].
+ssl_not_started(Config) when is_list(Config) ->
+    application:stop(ssl),
+    try
+        {error, ssl_not_started} = ssl:connect("www.erlang.org", 443, []),
```

Comment:
Should we not have a valid option list in the test, even if might fail before it is checked! So [{verify, none}] should be added!

Reply:
Yes, I have obviously not tested this on a 26 branch, that would fail.